nonprofit-board-management.php

```diff
@@ -946,6 +951,39 @@ private function validate_gravatar( $user_email ) {
 
       return $has_valid_avatar;
     }
+
+    /**
+     * Check if the Block Editor is being used.
+     * 
+     * @return bool True if the Block Editor is being used, false if not.
+     */
+    private function is_block_editor_in_use() {
+
+      $use_block_editor = get_option( 'classic-editor-replace' ) === 'no-replace';
```

Comment:
Can you give me an idea of what this does and whether we still need it?

Reply:
This should be removed, I moved this line here for testing to check the value before this function was returning - this same check is done further down in the function. The Classic Editor plugin has a setting under 'Writing' that allows you to set how the Classic Editor plugin works. You can either set it to completely replace the block editor or you can set it to use the block editor but add a 'Classic Editor' option to easily use the classic editor instead.

Reply:
This line has been removed.